.tav.yml

```
@@ -189,6 +189,12 @@ elasticsearch:
 handlebars:
   versions: '*'
   commands: node test/instrumentation/modules/handlebars.js
+jade:
+  versions: '*'
```

It looks like the API for Jade was significantly different in version 0.5.5 and below which is why the TAV tests are failing. Since this version is 8 years old, I vote for just not supporting that. In that case, just change this:

```suggestion
  versions: '>0.5.5'
```

---

I found it took a lot of time to test from 0.0.0 to 2.0.0., what about we just set jade >1.0.0 and pug >2.0.0 ？

---

We, unfortunately, have the same issue with all our other dependency tests. So far we've just accepted the time it took and limited our test-coverage to where we actually support it. So I vote to keep testing the entire supported range.